packages/ffe-tabs-react/src/TabButton.md

```diff
@@ -0,0 +1,44 @@
+# Tab buttons
```

Comment:
Styleguidist will render a header for us, so this should be removed (otherwise it will render as a duplicate h1)